src/refactor_nrepl/artifacts.clj

```diff
@@ -10,10 +10,23 @@
             [refactor-nrepl.ns.slam.hound.regrow :as slamhound-regrow]
             [version-clj.core :as versions])
   (:import java.util.Date
+           java.util.zip.GZIPInputStream
            java.util.jar.JarFile))
 
+(def artifacts-file ".artifacts-cache")
```

Comment:
I think we should move this to the platform's temp dir:

1. That way we can share it across all projects, independently of the dir the REPL was started in
2. It makes it unnecessary to add this file to the `.gitignore` of every project you work on or your git config.